napi-inl.h

```diff
@@ -425,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
+
+template <typename Key>
+class PropertyLValue {
```

Comment:
I might just name this `PropertyValue` and drop the `L`. But I don't feel strongly about it.